src/main/java/org/broadinstitute/hellbender/engine/filters/MappingQualityReadFilter.java

```diff
@@ -10,8 +10,10 @@
 /**
  * Keep reads with mapping qualities within a specified range.
  *
- * Note: this filter does not handle specially the unavailable mapping quality ({@link org.broadinstitute.hellbender.utils.QualityUtils#MAPPING_QUALITY_UNAVAILABLE}).
- * Use {@link org.broadinstitute.hellbender.engine.filters.ReadFilterLibrary.MappingQualityAvailableReadFilter} to explicitly filter out reads with unavailable quality.
+ * <p>Note: this filter is not designed to handle the unavailable mapping quality (255).
+ * Use MappingQualityAvailableReadFilter to explicitly filter out reads with unavailable quality.
+ * @see org.broadinstitute.hellbender.utils.QualityUtils#MAPPING_QUALITY_UNAVAILABLE
+ * @see org.broadinstitute.hellbender.engine.filters.ReadFilterLibrary.MappingQualityAvailableReadFilter
  */
 @DocumentedFeature(groupName= HelpConstants.DOC_CAT_READFILTERS, groupSummary=HelpConstants.DOC_CAT_READFILTERS_SUMMARY)
```

Comment:
@vdauwera A couple of things to think about. Since the @see links are javadoc links, they will surface in and be navigable from the *javadoc* for this filter. But I don't think they'll surface in the *gatkdoc*. I think thats a good thing, since it might be a bit confusing to send a GATK user into the middle of the javadoc, which looks and feels like programmer's doc. We probably could change the way these links work in the gatkdoc, but that would depend on making an assumption about the relative locations of the two doc sets. "extraDocs" links provide cross-links only between things that have @DocumentedFeature, so they avoid crossing this line. So I think it depends on how we want to organize the all and hat experience we want. Let me know what you think and we can figure out anything needs to change.